server/routes.js

```diff
@@ -0,0 +1,237 @@
+  }
+  return request.protocol + '://' + request.hostname +
+    (config && config.get('port') ? ':' + config.get('port') : '');
+}
+
+function makeCallback(response, next, type) {
+  return function(error, data) {
+    if (error !== null) {
+      next(error);
+    } else {
+      if (type === undefined) {
+        respond.success(response, data);
+      } else {
+        type(response, data);
+      }
+      next();
+    }
+  };
+}
+
+function makeMiddleware(handler, type) {
+  return function(request, response, next) {
+    handler(request, makeCallback(response, next, type));
+  };
+}
+
+function getUUID(request, callback) {
+  api.info.uuid(callback);
+}
+
+function isConnected(request, callback) {
+  api.info.isConnected(callback);
+}
+
+function getFee(request, callback) {
+  api.info.fee(callback);
+}
+
+function getServerStatus(request, callback) {
+  api.info.serverStatus(callback);
+}
+
+function generateWallet(request, callback) {
+  api.wallet.generate(callback);
+}
+
+function getAccountPayments(request, callback) {
+  const account = request.params.account;
+  const source_account = request.query.source_account;
+  const destination_account = request.query.destination_account;
+  const direction = request.query.direction;
+  const options = {
+    ledger_index_min: request.query.start_ledger,
+    ledger_index_max: request.query.end_ledger,
+    earliest_first: request.query.earliest_first === 'true',
+    exclude_failed: request.query.exclude_failed === 'true',
+    results_per_page: request.query.results_per_page,
+    page: request.query.page
+  };
+  api.payments.getAccountPayments(account, source_account, destination_account,
+    direction, options, callback);
+}
+
+function getPayment(request, callback) {
+  const account = request.params.account;
+  const identifier = request.params.identifier;
+  api.payments.get(account, identifier, callback);
+}
+
+function getPathFind(request, callback) {
+  const source_account = request.params.account;
+  const destination_account = request.params.destination_account;
+  const destination_amount_string = request.params.destination_amount_string;
+  const source_currency_strings = request.query.source_currencies;
+  api.payments.getPathFind(source_account, destination_account,
+    destination_amount_string, source_currency_strings, callback);
+}
+
+function getOrders(request, callback) {
+  const account = request.params.account;
+  const options = {
+    ledger: request.query.ledger,
+    limit: request.query.limit,
+    marker: request.query.marker
+  };
+  api.orders.getOrders(account, options, callback);
+}
+
+function getOrderBook(request, callback) {
+  const account = request.params.account;
+  const base = request.params.base;
+  const counter = request.params.counter;
+  const options = {
+    ledger: request.query.ledger,
+    limit: request.query.limit,
+    marker: request.query.marker
+  };
+  api.orders.getOrderBook(account, base, counter, options, callback);
```

Comment:
Since this PR is not meant to be a full refactoring, we should address this in a later PR. We're already circulating a proposal for a refactor of the high level API, and we can comment there.